src/parts/pintable.cpp

```diff
@@ -9557,8 +9562,7 @@ STDMETHODIMP PinTable::put_Gravity(float newVal)
       SetGravity(newVal);
       const float minSlope = (m_overridePhysics ? m_fOverrideMinSlope : m_angletiltMin);
       const float maxSlope = (m_overridePhysics ? m_fOverrideMaxSlope : m_angletiltMax);
-      const float slope = minSlope + (maxSlope - minSlope) * m_globalDifficulty;
-      g_pplayer->m_physics->SetGravity(slope, m_overridePhysics ? m_fOverrideGravityConstant : m_Gravity);
+      g_pplayer->m_physics->SetGravity(GetPlayfieldSlope(), m_overridePhysics ? m_fOverrideGravityConstant : m_Gravity);

```

Comment:
@Nicals Is this one here really correct? Cause seems like m_overridePhysics is ignored now.

Reply:
You are right, I did not knew this physics override settings. Should be fixed now, I clearly went too quickly on this one.

Reply:
Thanks!